editor/inserter/menu.js

```diff
@@ -58,37 +57,69 @@ class InserterMenu extends Component {
 		} );
 	}
 
-	selectBlock( name ) {
+	selectBlock( blockType ) {
```

Comment:
Because we used to store the whole block before that data was moved into the app state. Will change!